src/dolphin/os/init/__ppc_eabi_init.c

```diff
@@ -11,8 +11,8 @@ extern voidfunc __ctors[];
 
 static void __init_cpp(void);
 
-SECTION_INIT asm void __init_hardware(void)
-{ // clang-format off
+SECTION_INIT asm void __init_hardware(void){
+    // clang-format off
```

Comment:
idk why it keeps doing this

Reply:
clang-format is doing this? never had that happen for me